manifests/mg.pp

```diff
@@ -4,9 +4,11 @@
   $homedir = '/root',
   $default = true,
 ) {
+  contain epel
```

Comment:
Do we actually need to contain here? Seems like `include` should work equally well, since the package has a `require` statement, and that has less of a chance of dependency cycles.